src/components/magnet/magnet-util.ts

```diff
@@ -68,5 +66,5 @@ function getFieldForPointCharge(pointCharge: PointCharge, x: number, y: number)
   const distanceVec = pointVec.subtract(chargeVec);
   const distanceUnitVec = distanceVec.unit();
 
-  return distanceUnitVec.multiply(pointCharge.charge / (distanceVec.length() ** 2));
+  return distanceUnitVec.multiply(pointCharge.charge / Math.max(.1, (distanceVec.length() ** 2)));
```

Comment:
This and the changes below were hacks to deal with divide by zero errors...they're cleaned up once I add internal field line calculations